src/constants/contextProps/index.ts

```diff
@@ -0,0 +1,16 @@
+export type questionProps = {
+  id: string;
```

Comment:
Maybe, you can make it `_id` as you did in the below another type. Just to keep things consistent.